src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var maxX = window.innerWidth - this._margin.right;
+			var maxY = window.innerHeight - this._margin.bottom;
```

Comment:
I agree with @amatiasq that windows should be screen agnostic. This event triggered from the window manager:  https://github.com/rlamana/Ventus/blob/master/src/ventus/wm/window.js#L203 keeps the 'mousemove' event within WindowManager's boundaries. So, if any the WindowManager should be the one adapting to your desired space.